src/generator/AutoRest.Ruby.Azure/CodeGeneratorRba.cs

```diff
@@ -73,8 +72,11 @@ public CodeGeneratorRba()
                     continue;
                 }
 
-                var modelTemplate = new ModelTemplate { Model = model };
-                await Write(modelTemplate, Path.Combine(GeneratorSettingsRb.Instance.modelsPath, CodeNamer.UnderscoreCase(model.Name) + ImplementationFileExtension));
+                var modelTemplate = new AzureModelTemplate { Model = model };
+                if(!CompositeTypeRba.nameRegEx.IsMatch(model.Name.ToString()) || !CompositeTypeRba.IsResourceModelsMatchStandardDefinition(model))
```

Comment:
minor: a space after `if` and before `(` to keep the formatting consistent across codebase